db/migrate/20161229204445_set_default_content_for_posts.rb

```diff
@@ -0,0 +1,8 @@
+class SetDefaultContentForPosts < ActiveRecord::Migration
+  def up
+    change_column_default :posts, :content, ''
```

Comment:
If this is literally only useful when manually creating posts in the console, I'd rather skip the database migration, they're kind of a pain.